### PR TITLE
Adds changelog for 8.2.0

### PR DESCRIPTION
**Pull request by sruke**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,27 @@
...
+- Using FixedTimeEquals in NETCore targets. See [2857](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2857).
+- Updated .NET 9 to RC 2 [2898](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2898).
+- Adds ability to create token without kid [2968](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2968)
+- Enables code coverage in PRs [2946](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2946)
+- Various test improvements:
+- (#2953)[https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2953]
+- (#2955)[https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2955]
+- (#2951)[https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2951]
+- (#2952)[https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2952]
+- (#2947)[https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2947]
```
Comment (kllysng) [inline CHANGELOG.md]:
```suggestion
- [#2947](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2947)
```
And for the ones below 🙏

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,27 @@
...
+8.2.0
+=====
+### Fundamentals
+- Update System.Text.Json to 8.0.5 CVE-2024-43485. See [2892](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2892).
+- Using FixedTimeEquals in NETCore targets. See [2857](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2857).
+- Updated .NET 9 to RC 2 [2898](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2898).
+- Adds ability to create token without kid [2968](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2968)
+- Enables code coverage in PRs [2946](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2946)
+- Various test improvements:
+- (#2953)[https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2953]
```
Comment (kllysng) [inline CHANGELOG.md]:
```suggestion
- #2953
```
I think we can do this now 🤔 
Adding all the links takes a lot to work (and fixing all the ones below), might be worth testing if this works.

Comment (sruke) [reply CHANGELOG.md]: Done. Thank you for pointing it out Kelly.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,27 @@
...
+- Enables code coverage in PRs [2946](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2946)
+- Various test improvements:
+- [#2953](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2953)
+- [#2955](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2955)
+- [#2951](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2951)
+- [#2952](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2952)
+- [#2947](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2947)
+
+### Work related to redesign of IdentityModel's token validation logic [#2711](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/issues/2711)
+- Validates Audience for SAML2TokenHandler with New Model (2863)[https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2863]
```
Comment (kllysng) [inline CHANGELOG.md]: Sruthi I'm so sorry, I saw you pushed some fixes but did you see these ones too? And below, lines 19-23

**Review by kllysng — APPROVED** [review]
LGTM

**Review by jennyf19 — APPROVED** [review]
:shipit:

File: CHANGELOG.md (hunk 3 above)
Comment (sruke) [reply CHANGELOG.md]: I missed them in the initial commit and pushed another commit a few minutes later.

**Pull request merged**